index.bs

```diff
@@ -3861,6 +3864,45 @@ This [=registration extension=] and [=authentication extension=] enables use of
                     01           -- Subitem 3: CBOR short for Matcher Protection Type Software
     </pre>
 
+
+## Biometric Authenticator Performance Bounds Extension (biometricBound) ## {#sctn-authenticator-biometric-criteria-extension}
+
+This [=registration extension=] allows [=[RPS]=] to specify the desired performance bounds for selecting [=biometric authenticators=]
+as candidates to be employed in a [=registration=] ceremony. 
+
+: Extension identifier
+:: `biometricMatchingCriteria`
+
+: Client extension input
+:: Biometric performance bounds:
+
+    <pre class="idl">
+    dictionary authenticatorBiometricPerfBounds{
+        float FAR;
+        float FRR;
+        };
+    </pre>
+
+    The FAR is the maximum false acceptance rate for a biometric authenticator allowed by the [=[RP]=].
+
+    The FAR is the maximum false rejection rate for a biometric authenticator allowed by the [=[RP]=].
+
+: Client extension processing
+:: This extension can only be used during {{CredentialsContainer/create()}}. If the client supports the this
+    extension and at least one [=biometric authenticator=] is available, it MUST use the first available biometric authenticator whose FAR and FRR match the bounds as provided.
```

Comment:
First off: s/the this/this/

---

Second:

>If the client supports the this extension and at least one biometric authenticator is available, it MUST use the first available biometric authenticator whose FAR and FRR match the bounds as provided.

This formulation would likely be confusing with the new hot-plugging approach for `create()`. We no longer assume that there will be a single instant when you can list all "available" authenticators. Instead they will appear at some point (possibly immediately, e.g., platform authenticators) during the timeout, and the client can't know if the user will plug in an eligible biometric authenticator after the client has found a non-biometric authenticator. I therefore suggest reformulating this to something more like:

>If the client supports this extension, it MUST NOT use a biometric authenticator whose FAR or FRR does not match the bounds as provided.